geometry_msgs/Pose position
bool isTagVisible